src/test/ui/in-band-lifetimes/elided-lifetimes.rs

```diff
@@ -0,0 +1,45 @@
+// Copyright 2018 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+// run-rustfix
+// compile-flags: --edition 2018
+
+#![allow(unused)]
+#![deny(elided_lifetimes_in_paths)]
+//~^ NOTE lint level defined here
+
+use std::cell::{RefCell, Ref};
```

Comment:
Can we test:

- [ ] some case with more than one parameter (just for fun, though I don't see why it wouldn't work)
- [ ] a case where there is more than one lifetime and you only supply *some*?
- [ ] the macro case where the type is constructed in a macro from bits and pieces?